tests/core/arch-arm64/arm64stalker.c

```diff
@@ -0,0 +1,652 @@
+/*
+ * Copyright (C) 2009-2014 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ * Copyright (C) 2010-2013 Karl Trygve Kalleberg <karltk@boblycat.org>
+ *
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "arm64stalker-fixture.c"
+
+TEST_LIST_BEGIN (arm64stalker)
+
+/* EVENTS */
+STALKER_ARM64_TESTENTRY (no_events)
+STALKER_ARM64_TESTENTRY (call)
+STALKER_ARM64_TESTENTRY (ret)
+STALKER_ARM64_TESTENTRY (exec)
+
+/* BRANCH */
+STALKER_ARM64_TESTENTRY (unconditional_branch)
+STALKER_ARM64_TESTENTRY (unconditional_branch_reg)
+STALKER_ARM64_TESTENTRY (conditional_branch)
+STALKER_ARM64_TESTENTRY (compare_and_branch)
+STALKER_ARM64_TESTENTRY (test_bit_and_branch)
+
+/* FOLLOWS */
+STALKER_ARM64_TESTENTRY (follow_std_call)
+STALKER_ARM64_TESTENTRY (follow_return)
+STALKER_ARM64_TESTENTRY (follow_syscall)
+STALKER_ARM64_TESTENTRY (follow_thread)
+
+/* EXTRA */
+STALKER_ARM64_TESTENTRY (heap_api)
+STALKER_ARM64_TESTENTRY (no_register_clobber)
```

Comment:
These should be indented. Let's also drop the `_ARM64` part, there will only be one Stalker test-suite compiled into the binary at any point as they're arch-dependent.